<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela_Clientes_Consulta</class>
 <widget class="QMainWindow" name="Janela_Clientes_Consulta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Consulta Clientes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 225, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblRg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>232</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RG:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>331</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="txtNomeClienteConsulta">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: groove;
border-width: 1px;
border-color: rgb(204, 204, 204)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSobreNomeClienteConsulta">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: groove;
border-width: 1px;
border-color: rgb(204, 204, 204)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRgClienteConsulta">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: groove;
border-width: 1px;
border-color: rgb(204, 204, 204)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_EnderecoConsulta">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: groove;
border-width: 1px;
border-color: rgb(204, 204, 204)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblNomeCl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>172</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSobreNome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>202</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sobre Nome:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultaIdCliente">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>95</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;
border-style: groove;
border-width: 2px
</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CONSULTA DE CLIENTES</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEndereco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>262</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Endere√ßo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNome_BuscaIdCl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Busca por ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNome_BuscaNomCl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Busca por Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtconsultaClienteId">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: groove;
border-width: 1px;
border-color: rgb(204, 204, 204)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtconsultaClienteNome">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: groove;
border-width: 1px;
border-color: rgb(204, 204, 204)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
